release/ray_release/cluster_manager/full.py

```diff
@@ -91,6 +91,11 @@ def start_cluster(self, timeout: float = 600.0):
             )
             completed = result.result.completed
 
+        # Sleep 30 seconds to let everything come up (this is a short-term hack to see if
+        # we can get release tests working on the new stack, since it seems like session
+        # state operations are not indicative of the cluster being externally available).
+        time.sleep(30)
```

Comment:
should not be necessary as we have wait for nodes - it just needs a shorter timeout